### PR TITLE
cloud: add ui access docs

### PR DESCRIPTION
**Pull request by michaellzc**
DO NOT MERGE until Entitle is announced by #security officially

### CODE REVIEW

**Comment by michaellzc** [comment]
Current dependencies on/for this PR:
* main
  * **PR #5715** <a href="https://app.graphite.dev/github/pr/sourcegraph/handbook/5715" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/sourcegraph/handbook/5715?utm_source=stack-comment).

File: content/departments/cloud/technical-docs/oidc_site_admin.md [hunk 1]
```diff
@@ -1,31 +1,66 @@
...
 
-2. All [Cloud Team members](https://github.com/sourcegraph/infrastructure/blob/main/security/managed-instance-okta/variables.tf#L1)
-3. Additonal Customer Engineeger(s) from [CE List](https://github.com/sourcegraph/infrastructure/blob/main/security/managed-instance-okta/variables.tf#L13) added to `config.yaml` via `additionalAdmins` - [sample](https://github.com/sourcegraph/deploy-sourcegraph-managed/blob/7e9066e537b02feb6013585d443fc27514b71a71/addepar/config.yaml#L5)
+To login to the Cloud instance UI, open `https://company.sourcegraph.com/sign-in?sourcegraph-operator`, and click `Continoue with Sourcegraph Operator` to authenticate.
 
 ## Disabling OKTA OIDC on Managed Instance
 
-OIDC OKTA access is disabled on instance when explictly asked by customer via configuration flag `disableSourcegraphManagementAccess: true` in `config.yaml`.
+SOAP access cane be disabled on instance when explictly asked by customer via configuration flag `disableSourcegraphManagementAccess: true` in `config.yaml`.
```
Comment (bobheadxi) [inline content/departments/cloud/technical-docs/oidc_site_admin.md]:
```suggestion
SOAP access can be disabled on an instance when explictly asked to by a customer via the configuration flag `disableSourcegraphManagementAccess: true` in `config.yaml`.
```

**Comment by unknwon** [comment]
FYI: Converted to "Draft" so no one can hit "Merge" by accident.

File: content/departments/cloud/index.md [hunk 2]
```diff
@@ -166,16 +166,16 @@ Incidents which affect managed instances are handled according to our [incidents
...
+| Login to customer MI web UI | Cloud/CE/CS/Engg                                      | Site admin access to customer instance                                                                           | [Sourcegraph teammate access to Cloud instances](./technical-docs/oidc_site_admin.md)                                                                                                                                                                                               |
+| Login to customer Grafana   | Cloud/CE/CS/Engg                                      | DONT, use centralized observability                                                                              | learn more from [centralized o11y](./technical-docs/observability/index.md#common-operations)                                                                                                                                                                                       |
```
Comment (unknwon) [inline content/departments/cloud/index.md]: "Eng" or "Engg"? What's the latter?

File: content/departments/cloud/technical-docs/observability/index.md [hunk 3]
```diff
@@ -22,6 +22,7 @@ To opt-in a panel to this multi-instance dashboard, see [how to add monitoring](
 
 ### Common operations
 
+- Request access to the Grafana dashboard, follow [Requesting access to Grafana](./operations.md#requesting-access-to-grafana)
```
Comment (unknwon) [inline content/departments/cloud/technical-docs/observability/index.md]:
```suggestion
- Request access to the Grafana dashboard, follow [Requesting access to Grafana](./operations.md#requesting-access-to-grafana).
```

File: content/departments/cloud/technical-docs/oidc_site_admin.md [hunk 4]
```diff
@@ -1,31 +1,66 @@
-# Using OpenID Connect for Site Admin Access
+# Sourcegraph teammate access to Cloud instances
 
-Sourcegraph employees access to managed instances application interface (Web UI) is restricted to essential personnel only. This ensures Sourcegraph is able to help customers troubleshoot issues and deliver a smooth experience. We utilize OpenID Connect to enable Sourcegraph employees access to customer instance to make sure there is an audit trail for every access.
+Sourcegraph temmates access to Cloud instances application interface (Web UI) is restricted to essential personnel only. This ensures Sourcegraph is able to help customers troubleshoot issues and deliver a smooth experience. We utilize Sourcegraph Operator Authentication Provider (SOAP) which implements OpenID Connect to enable Sourcegraph employees access to customer instance to make sure there is an audit trail for every access.
```
Comment (unknwon) [inline content/departments/cloud/technical-docs/oidc_site_admin.md]:
```suggestion
Sourcegraph teammates access to Cloud instances application interface (Web UI) is restricted to essential personnel only. This ensures Sourcegraph is able to help customers troubleshoot issues and deliver a smooth experience. We utilize Sourcegraph Operator Authentication Provider (SOAP) which implements OpenID Connect to enable Sourcegraph employees access to customer instance to make sure there is an audit trail for every access.
```

File: content/departments/cloud/technical-docs/oidc_site_admin.md [hunk 5]
```diff
@@ -1,31 +1,66 @@
...
 
-## Enable OIDC for Managed Instance
+The Cloud team manages a separate Cloud OKTA account (separate from the company-wide Sourcegraph OKTA account) to manage UI access to Cloud instances. The Cloud OKTA account is federated by the parent Sourcegraph OKTA account to ensure access control is consistent across all our systems. For example, if an account is deactivated from the Sourcegraph OKTA account, the user will loss access to Cloud OKTA as well.
```
Comment (unknwon) [inline content/departments/cloud/technical-docs/oidc_site_admin.md]: I believe the correct casing is "Okta"

File: content/departments/cloud/technical-docs/oidc_site_admin.md [hunk 6]
```diff
@@ -1,31 +1,66 @@
...
 
-Unless customer explicitly [disabled OIDC on Managed Instance](#disabling-okta-oidc-on-managed-instance)
+> Consider installing [requestly](https://requestly.io/) and use this [rule](https://app.requestly.io/rules/#sharedList/1670019946529-Michael-shared-list-12-2-2022) to automatically append the `sourcegraph-operator` query param on any instance signin page.
 
-2. All [Cloud Team members](https://github.com/sourcegraph/infrastructure/blob/main/security/managed-instance-okta/variables.tf#L1)
-3. Additonal Customer Engineeger(s) from [CE List](https://github.com/sourcegraph/infrastructure/blob/main/security/managed-instance-okta/variables.tf#L13) added to `config.yaml` via `additionalAdmins` - [sample](https://github.com/sourcegraph/deploy-sourcegraph-managed/blob/7e9066e537b02feb6013585d443fc27514b71a71/addepar/config.yaml#L5)
+To login to the Cloud instance UI, open `https://company.sourcegraph.com/sign-in?sourcegraph-operator`, and click `Continoue with Sourcegraph Operator` to authenticate.
 
 ## Disabling OKTA OIDC on Managed Instance
```
Comment (unknwon) [inline content/departments/cloud/technical-docs/oidc_site_admin.md]: I think this section should be now called "Disabling Sourcegraph management access", WDYT?

File: content/departments/cloud/technical-docs/oidc_site_admin.md [hunk 7]
```diff
@@ -1,31 +1,66 @@
...
 
-OIDC OKTA access is disabled on instance when explictly asked by customer via configuration flag `disableSourcegraphManagementAccess: true` in `config.yaml`.
+SOAP access can be disabled on an instance when explictly asked to by a customer via the configuration flag `disableSourcegraphManagementAccess: true` in `config.yaml`.
 
 ## FAQ
 
 ### Does it affect the number of seats customers pay for in the license?
 
-Every essential Sourcegraph personnel will effectively be an actual user in the customer instance, so they will be counted toward in the license seat count. However, we allocated addition 10 seats in the license to accommodate the seats used by internal Sourcegraph teammates with Site Admin access.
+No, users login through SOAP is a special account type, and they are excluded from license count.
```
Comment (unknwon) [inline content/departments/cloud/technical-docs/oidc_site_admin.md:71]: This line makes the PR also dependent on the closure of https://github.com/sourcegraph/customer/issues/1432, when we stop using legacy OpenID for all MIs for both CEs and Cloud team.

**Review by unknwon — COMMENTED** [review]
Thanks for the doc update!

File: content/departments/cloud/technical-docs/oidc_site_admin.md [hunk 8]
```diff
@@ -1,31 +1,66 @@
...
 
-Unless customer explicitly [disabled OIDC on Managed Instance](#disabling-okta-oidc-on-managed-instance)
+> Consider installing [requestly](https://requestly.io/) and use this [rule](https://app.requestly.io/rules/#sharedList/1670019946529-Michael-shared-list-12-2-2022) to automatically append the `sourcegraph-operator` query param on any instance signin page.
 
-2. All [Cloud Team members](https://github.com/sourcegraph/infrastructure/blob/main/security/managed-instance-okta/variables.tf#L1)
-3. Additonal Customer Engineeger(s) from [CE List](https://github.com/sourcegraph/infrastructure/blob/main/security/managed-instance-okta/variables.tf#L13) added to `config.yaml` via `additionalAdmins` - [sample](https://github.com/sourcegraph/deploy-sourcegraph-managed/blob/7e9066e537b02feb6013585d443fc27514b71a71/addepar/config.yaml#L5)
+To login to the Cloud instance UI, open `https://company.sourcegraph.com/sign-in?sourcegraph-operator`, and click `Continoue with Sourcegraph Operator` to authenticate.
 
 ## Disabling OKTA OIDC on Managed Instance
```
Comment (michaellzc) [inline content/departments/cloud/technical-docs/oidc_site_admin.md]:
```suggestion
## Disabling Sourcegraph management access
```

**Pull request merged**